jekyll/_cci2/deploy-to-amazon-sagemaker.adoc

```diff
@@ -0,0 +1,368 @@
+. Enter your "Provider URL". This will be `\https://oidc.circleci.com/org/<your-organization-id>`, where `<your-organization-id>` is the ID of your CircleCI organization.
++
+{% include snippets/find-organization-id.adoc %}
+. Click **Get Thumbprint**
+. For "Audience" enter your organization ID
+. Click btn:[Add Provider]
+
+Next, create a Policy and then a Role.
+
+[#policy]
+=== b. Policy
+
+. From your AWS management console, navigate to menu:IAM[Policies]
+. Click btn:[Create Policy]
+. Click btn:[JSON] to select the JSON policy editor and then copy in the following permissions. We have organized the permissions into two groups. OrbPermissions and S3Access statements are used for the deployment of the model to the endpoints. The S3AccessTrainModel and SageMakerTrainModel statements are needed if you want to train the demo model we provide.
++
+NOTE: Update the S3 bucket information to match your setup. Create a bucket if one does not yet exist.
++
+[,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "OrbPermissions",
+            "Effect": "Allow",
+            "Action": [
+                "sagemaker:AddTags",
+                "sagemaker:CreateEndpointConfig",
+                "sagemaker:CreateModel",
+                "sagemaker:DescribeEndpoint",
+	 	        "sagemaker:DescribeEndpointConfig",
+                "sagemaker:ListEndpoints",
+                "sagemaker:ListModelPackages",
+                "sagemaker:ListTags",
+                "sagemaker:UpdateEndpoint",
+                "iam:PassRole"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "S3Access",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:ListBucket"
+            ],
+            "Resource": [
+                "arn:aws:s3:::<BUCKET_FOR_MODEL_ASSETS>/*"
+            ]
+        },
+        {
+            "Sid": "S3AccessTrainModel",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:ListBucket",
+                "s3:PutObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::sagemaker-sample-files/*",
+                "arn:aws:s3:::circleci-sagemaker-pipeline",
+                "arn:aws:s3:::circleci-sagemaker-pipeline/*"
+            ]
+        },
+        {
+            "Sid": "SageMakerTrainModel",
+            "Effect": "Allow",
+            "Action": [
+                "sagemaker:CreateTrainingJob",
+                "sagemaker:DescribeTrainingJob",
+                "logs:DescribeLogStreams",
+                "sagemaker:ListModelPackageGroups",
+                "sagemaker:CreateModelPackage",
+                "sagemaker:UpdateModelPackage"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
+
+. Scroll down and click btn:[Next]
+. Give your policy a name and then click btn:[Create Policy]
+
+[#role]
+=== c. Role
```

Comment:
I added extra steps to this section to be complete, needs checking

Reply:
@meeech I'm not sure I'm able to go through all steps in AWS without messing up things - are you able to review this part?

Reply:
yeah, eyeballing it now - it  looks fine. I'll work to find some time to run though it tomorrow properly - to actually do it. I wont have a chance today in the end.